src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/Encoder.cs

```diff
@@ -15,7 +15,7 @@ internal sealed class Encoder
         private const int MinQuality = 0;
         private const int MaxQuality = 11;
         private bool _isDisposed = false;
-        internal IntPtr State = IntPtr.Zero;
+        internal IntPtr _state = IntPtr.Zero;
```

Comment:
Again, if you are exposing these for external usage outside the class scope, these should be modeled at read-only properties.